Sources/Basics/Vendor/Triple.swift

```diff
@@ -426,7 +426,7 @@ extension Triple {
     }
   }
 
-  public enum Arch: String, CaseIterable {
+  public enum Arch: String, CaseIterable, Decodable {
```

Comment:
This also needs to be upstreamed to swift-driver, right?

Reply:
Added in https://github.com/apple/swift-driver/pull/1512